uniflow/op/model/cv/model_server.py

```diff
@@ -262,3 +268,204 @@ def __call__(self, data: List[str]) -> List[str]:
             out = re.sub(r"\n{3,}", "\n\n", final_md.strip()).strip()
             outs.append(out)
         return outs
+
+
+class LlamaParseModelServer(AbsModelServer):
+    """Llama Parse Model Server"""
+
+    def __init__(
+        self, prompt_template: PromptTemplate, model_config: Dict[str, Any]
+    ) -> None:
+        super().__init__(prompt_template, model_config)
+        self._model_config = LlamaParseModelConfig(**self._model_config)
+
+        try:
+            from llama_parse import (  # pylint: disable=import-outside-toplevel
+                LlamaParse,
+            )
+
+        except ModuleNotFoundError as exc:
+            raise ModuleNotFoundError(
+                "Please install llama_parse to use LlamaModelServer. You can use `pip install llama_parse` to install it."
+            ) from exc
+
+    def _preprocess(self, data: str) -> List[str]:
+        """Preprocess data.
+
+        Args:
+            data (List[str]): Data to preprocess.
+
+        Returns:
+            List[str]: Preprocessed data.
+        """
+        return data
+
+    def _postprocess(self, data: List[str]) -> List[str]:
+        """Postprocess data.
+
+        Args:
+            data (List[str]): Data to postprocess.
+
+        Returns:
+            List[str]: Postprocessed data.
+        """
+        return data
+
+    def __call__(self, data: List[str]) -> List[str]:
+        """Run model.
+
+        Args:
+            data (List[str]): Data to run.
+
+        Returns:
+            List[str]: Output data.
+        """
+        from llama_parse import LlamaParse
+
+        parser = LlamaParse(
+            api_key=self._model_config.api_key,
+            result_type=self._model_config.result_type,  # "markdown" and "text" are available
+            num_workers=self._model_config.num_wokers,  # if multiple files passed, split in `num_workers` API calls
+            verbose=True,
+            language=self._model_config.language,  # Optionally you can define a language, default=en
+        )
+
+        out = []
+
+        if self._model_config.sync:
+            for pdf in data:
+                result = parser.load_data(pdf)
+                out.append(result[0].text)
```

Comment:
qq: is the result always a list of size 1. In other words, can it pass in a list of pdf and return a list of result?